pkg/js/parse_tests/029-dextendsub/düsseldorf.example.net.zone

```diff
@@ -0,0 +1,9 @@
+$TTL 300
+@                IN A     10.0.0.7
+düsseltal       IN A     10.0.0.11
+www.düsseltal   IN A     10.0.0.12
+subdomain        IN A     10.0.0.9
+www.subdomain    IN A     10.0.0.10
+www              IN A     10.0.0.8
+ü               IN A     10.0.0.13
```

Comment:
Should a Unicode character like that appear in the zone file? I would think it should be punycode instead (but I confess I'm no expert in this area)

Reply:
Actually, I really don't know. Luckily, I haven't dealt with zone files in years :D

Would the test code load the correct zone file then? I'd expect a change to it to be necessary then, but it's certainly possible.